<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>resetPasswordWindow</class>
 <widget class="QMainWindow" name="resetPasswordWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">#centralwidget{
border-image: url(:/image/login/图片3.png);
}</string>
   </property>
   <widget class="QLabel" name="oldpassword">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>155</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;黑体&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>旧 密 码</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="usernamelineEdit">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>85</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/login/图片4.png);
font: 12pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>请输入用户名</string>
    </property>
   </widget>
   <widget class="QLabel" name="usernaeme">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>90</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;黑体&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>用 户 名</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="oldpasswordlineEdit">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>150</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/login/图片4.png);
font: 12pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>请输入旧密码</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="newpasswordlineEdit">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>215</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/login/图片4.png);
font: 12pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>请输入新密码</string>
    </property>
   </widget>
   <widget class="QLabel" name="newpassword">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>220</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;黑体&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>新 密 码</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ackpasswordlineEdit">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>280</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/login/图片4.png);
font: 12pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>请再次输入新密码</string>
    </property>
   </widget>
   <widget class="QLabel" name="ackpassword">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>285</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;黑体&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>确认密码</string>
    </property>
   </widget>
   <widget class="QPushButton" name="commitButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>350</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-image: url(:/image/login/图片4.png);
font: 10pt &quot;微软雅黑&quot;;
}
QPushButton:pressed{
font: 9pt &quot;微软雅黑&quot;;
}</string>
    </property>
    <property name="text">
     <string>确认</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
